actions/transfer-case.vbs

```diff
@@ -138,466 +178,296 @@ EMReadScreen mail_addr_line_four, 43, 12, 27
+      EditBox 205, 235, 20, 15, hc_cfr_year
+      ButtonGroup ButtonPressed
+        OkButton 235, 260, 50, 15
+        CancelButton 290, 260, 50, 15
+        PushButton 135, 5, 65, 15, "SPEC/XFER", XFER_button
+        PushButton 205, 5, 65, 15, "POLI/TEMP", POLI_TEMP_button
+        PushButton 275, 5, 65, 15, "USEFORM", useform_xfer_button
+      Text 5, 10, 60, 10, "Client Move Date"
+      Text 5, 30, 55, 10, "Excluded time?"
+      Text 140, 30, 40, 10, "Begin Date:"
+      Text 5, 50, 75, 10, "METS Case Number:"
+      Text 140, 50, 65, 10, "METS Case Status:"
+      Text 5, 70, 70, 10, "Reason For Transfer:"
+      Text 5, 90, 70, 10, "Actions To Be Taken:"
+      Text 5, 110, 135, 10, "List All Requested/Pending Verifications:"
+      Text 5, 130, 195, 10, "Note any expected changes in household's circumstances:"
+      Text 5, 150, 45, 10, "Other Notes:"
+      GroupBox 5, 165, 335, 90, "Current Financial Responsibility County (CFR)"
+      Text 95, 240, 75, 10, "Change Date (MM YY)"
+      Text 20, 200, 45, 10, "Current CFR:"
+      Text 10, 285, 45, 10, "Current CFR:"
+      Text 85, 285, 75, 10, "Change Date (MM YY)"
+      Text 20, 240, 45, 10, "Current CFR:"
+      Text 95, 200, 75, 10, "Change Date (MM YY)"
+    EndDialog
 
+	Do
+		Do
 			err_msg = ""
-			dialog Dialog1
+			Dialog Dialog1
 			cancel_confirmation
-			IF excluded_time = "Yes" AND isdate(excluded_date) = FALSE THEN MsgBox "Please enter a valid date for the start of excluded time or double check that the client's absense is due to excluded time."
-			IF isdate(move_date) = FALSE THEN MsgBox "Please enter a valid date for client move."
-			IF ucase(left(servicing_worker, 4)) = ucase(transferring_worker_county_code) THEN MsgBox "You must use the ''Within the Agency'' script to transfer the case within the agency. The Worker/Agency you have selected indicates you are trying to transfer within your agency."
-			IF (hc_status_check = "ACTV" AND excluded_time = "") THEN MsgBox "Please select whether the client is on excluded time."
-			IF manual_cfr_cash_checkbox = CHECKED AND cash_cfr_no_change_checkbox = CHECKED THEN MsgBox ("Please select whether the CFR for CASH is changing or not. Review input.")
-			IF manual_cfr_hc_checkbox = CHECKED AND hc_cfr_no_change_checkbox = CHECKED THEN MsgBOx ("Please select whether the CFR for HC is changing or not. Review input.")
-			If mets_active_checkbox = CHECKED and METS_case_number = "" then err_msg = err_msg & vbNewLine & "* Please enter a METS case number."
-			If err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbNewLine & err_msg & vbNewLine		'error message including instruction on what needs to be fixed from each mandatory field if incorrect
-		LOOP UNTIL err_msg = ""									'loops until all errors are resolved
+            IF ButtonPressed = POLI_TEMP_button THEN CALL view_poli_temp("TE02", "08", "095", "") 'TE02.08.095' there is no forth variable
+            IF ButtonPressed = XFER_button Then CALL MAXIS_dialog_navigation()
```

Comment:
The If statement isn't needed here, unless you are having some dialog display issues.

Reply:
updated and moved dialog to correct